frontend/src/pages/_recruitment/admin/RevokeAccessDialog.js

```diff
@@ -10,23 +10,37 @@ import {
     CircularProgress,
 } from '@material-ui/core'
 
-import { useDispatch } from 'react-redux'
+import { useSelector, useDispatch } from 'react-redux'
 
 import Button from 'components/generic/Button'
 import { useTranslation } from 'react-i18next'
 
 import * as RecruitmentActions from 'redux/recruitment/actions'
 import * as SnackbarActions from 'redux/snackbar/actions'
+import * as RecruitmentSelectors from 'redux/recruitment/selectors'
+import * as OrganiserActions from 'redux/organiser/actions'
 
 export default ({ userId, onClose }) => {
     const { t } = useTranslation()
     const dispatch = useDispatch()
     const [loading, setLoading] = useState(false)
+    const events = useSelector(RecruitmentSelectors.events)
 
     const handleRevokeAccess = useCallback(async () => {
         setLoading(true)
         try {
```

Comment:
- [ ] Optional: Consider using only one type of async code declaration, between:

try/catch
async/await
then/catch/finally

Reply:
noted for the future, will not fix now because it is admin side